README.md

```diff
@@ -158,6 +158,21 @@ Whether the system is connected to Insights; valid values are `present`
 Whether the system automatically updates the dynamic configuration. It is
 enabled by default.
 
+```yaml
+    rhc_insights:
+      ansible_host: "example-host"
+```
+
+Configures the ansible host name with a custom value for the system record
+in Host Based Inventory (HBI). This host name is used in playbooks by remediations.
+It is `null` by default and will use the system host name if not specified.
+
+Possible values of this variable:
+
+* `null` or an empty value: the ansible host name is not changed.
+* `{state: absent}`: the ansible host name is unset in the insights-client config file and Host Based Inventory (HBI) is updated to use the system host name.
```

Comment:
If we decided that an empty string means the same as `{state: absent}`:

```suggestion
* `{state: absent}` or an empty string: the ansible host name is unset in the insights-client config file and Host Based Inventory (HBI) is updated to use the system host name.
```

I don’t prefer this solution though.

Reply:
For some history about why we use the seemingly unintuitive `{state: absent}` - see https://linux-system-roles.github.io/documentation/incremental_settings.html

Also, our usage of `omit` is problematic - `omit` should almost never be used except for as a direct argument to a module parameter - users should certainly not be setting role parameters to the value `omit` - I would prefer not to mention it.